File: 5-Problem-A-Day/58. Length of Last Word.java
class Solution {
    public int lengthOfLastWord(String s) {
        int ans = 0;
        for (int i = 0; i < s.length(); i++) {
            if (i != 0 && s.charAt(i) != ' ' && s.charAt(i - 1) == ' ')
                ans = 1;
            else if (s.charAt(i) != ' ')
                ans += 1;
        }
        return ans;
    }
}

File: 2024/58. Length of Last Word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int i = s.length() - 1;
        while (i >= 0 && s[i] == ' ') {
            i -= 1;
        }
        int end = i;
        while (i >= 0) {
            if (s[i] == ' ') {
                break;
            }
            i--;
        }
        return end - i;
    }
};